modules/src/ics07_tendermint/consensus_state.rs

```diff
@@ -78,6 +83,15 @@ impl ConsensusState {
             next_validators_hash,
         }
     }
+
+    pub fn new_from_header(header: SignedHeader<TMCommit, TMHeader>) -> Self {
```

Comment:
Minor: maybe we can use the `From` trait to implement this? Not sure what the trade-off is.